tests/parser/AquiferTests.cpp

```diff
@@ -566,6 +566,104 @@ PORO
     BOOST_CHECK( conf == conf2 );
 }
 
+BOOST_AUTO_TEST_CASE(Test_Aquifer_Config_Active)
+{
+    const auto deck = Parser{}.parseString(R"(
+START             -- 0
+10 MAI 2007 /
```

Comment:
MAY?

Reply:
> MAY?

Yes, that's probably best.  On the one hand, both ECLIPSE and we support the Scandinavian spelling `MAI` to denote the [5th month of the year](https://github.com/OPM/opm-common/blob/98b49964f580edd16ce979b94d05586068c85ff6/src/opm/common/utility/TimeService.cpp#L35-L36), but for an international audience it's better to use the English spelling.  I'll update it.